Submodule lxi-tools contains modified content
diff --git a/lxi-tools/src/lxi_gui-window.c b/lxi-tools/src/lxi_gui-window.c
index 903f128..e7e776c 100644
--- a/lxi-tools/src/lxi_gui-window.c
+++ b/lxi-tools/src/lxi_gui-window.c
@@ -332,7 +332,7 @@ static void list_add_instrument (LxiGuiWindow *self, const char *name, const cha
     gtk_widget_add_css_class(list_subtitle, "subtitle");
 
     GtkCssProvider *provider = gtk_css_provider_new();
-    gtk_css_provider_load_from_string(provider, "label.subtitle {opacity: 1; font-size: x-small;}");
+    gtk_css_provider_load_from_string(provider, "label.subtitle {opacity: 1; font-size: small;}");
     GdkDisplay *display = gdk_display_get_default();
     gtk_style_context_add_provider_for_display(display, GTK_STYLE_PROVIDER(provider), GTK_STYLE_PROVIDER_PRIORITY_USER);
     gtk_widget_add_css_class(GTK_WIDGET(list_subtitle), "subtitle");
